src/core/Wizard/Step.tsx

```diff
@@ -40,6 +46,7 @@ export const Step = ({
   totalSteps,
   type,
   onClick,
+  description: tooltip,
```

Comment:
I think `description` is ok.

Reply:
Yeah, I don't know why I used different name from prop here. I went with description in latest commit.